eng/common/scripts/Cadl-Project-Sync.ps1

```diff
@@ -44,10 +44,10 @@ function GetGitRemoteValue([string]$repo) {
         $gitRemotes = (git remote -v)
         foreach ($remote in $gitRemotes) {
             if ($remote.StartsWith("origin")) {
-                if ($remote -match 'https://github.com/\S+[\.git]') {
+                if ($remote -match 'https://github.com/\S+') {
```

Comment:
What scenarios do we not have the ".git" at the end? I know it is supported but the majority of workflows folks use should have it. Instead of removing perhaps we make it optional `(\.git)?`. I'm actually not sure the older expression was doing what we wanted anyway as it is just matching a single char in the set `[\.git]`.

Reply:
I think the older expression was intended to be optional, but I used `[ ]` mistakenly instead of `( )` and due to the nature of of the difference it still passed testing

Reply:
Actually investigating further I think what you have probably works for the goal of this function which is really only to determine if you are using https or git protocol. We probably don't even need to use regex for that and could just do `$remote.StartsWith("...")` however what is here also works.